### PR TITLE
[Plugin Profile] Format x-axis as integers and add captions.

### PR DESCRIPTION
**Pull request by trisolaran**
make x-axis formatting as integer for vz-line charts
add axis caption for both x-y.

### CODE REVIEW

File: tensorboard/plugins/profile/input_pipeline_analyzer/input-pipeline-analyzer.html [hunk 1]
```diff
@@ -54,15 +54,28 @@
...
+      .x-axis-title {
+        text-align:center;
+      }
+      .y-axis-title {
+        writing-mode:bt-rl;
+        -webkit-transform:rotate(270deg);
+        -moz-transform:rotate(270deg);
+        -o-transform: rotate(270deg);
+        -ms-transform:rotate(270deg);
+        transform: rotate(270deg);
```
Comment (chihuahua) [inline tensorboard/plugins/profile/input_pipeline_analyzer/input-pipeline-analyzer.html]: Could we just use `transform` and remove the other platform-specific properties? Firefox and Chrome both support `transform`.

File: tensorboard/plugins/profile/input_pipeline_analyzer/input-pipeline-analyzer.html [hunk 2]
```diff
@@ -54,15 +54,28 @@
...
       }
       .table-style {
         table-layout:fixed;
         width:95%;
       }
       .performance-table {
-        width: 100%;
+        width:100%;
+      }
+      .x-axis-title {
```
Comment (chihuahua) [inline tensorboard/plugins/profile/input_pipeline_analyzer/input-pipeline-analyzer.html]: For both `x-axis-title` and `y-axis-title`, can we specify `text-transform: capitalize;`? That looks a bit more polished IMO. :)

Comment (trisolaran) [reply tensorboard/plugins/profile/input_pipeline_analyzer/input-pipeline-analyzer.html]: done. thanks for the tips

File: tensorboard/plugins/profile/input_pipeline_analyzer/input-pipeline-analyzer.html (hunk 1 above)
Comment (trisolaran) [reply tensorboard/plugins/profile/input_pipeline_analyzer/input-pipeline-analyzer.html]: done.

**Pull request merged**